articles/npm-packages/index.md

```diff
@@ -0,0 +1,191 @@
+
+<!--more-->
+
+### Table of Contents
+
+- [Introduction](#introduction)
+- [Create an NPM Account](#create-an-npm-account)
+- [Let's Code](#let's-code)
+- [Deployment](#deployment)
+- [Using the Package](#using-the-package)
+- [Further Reading](#further-reading)
+
+### Introduction
+
+A package manager is a software that manages versions, packages and dependencies for your project. When a project grows in scale, a package manager helps to manage dependencies. It works perfectly across versions and even in collaborative projects. It cuts costs by reducing dependency problems across developer teams.
+
+To install Node.js packages, we use the [NPM registry](https://www.npmjs.com/), which is a centralized repository of packages. In this article we will learn how to create a Node.js package and deploy it on the NPM registry for public use.
+
+For more background on Node.js and various related technologies, check out some amazing [Section.io](https://www.section.io/) articles below:
+
+- [History of Node.js](/engineering-education/history-of-nodejs/)
+- [Most Useful Node.js Packages](/engineering-education/most-useful-nodejs-packages/)
+- [Building a Node.js Application Using Docker](/engineering-education/building-a-nodejs-application-using-docker/)
+- [Linting in Node.js using ESLint](/engineering-education/node-eslint/)
+- [APIs in Node.js vs Python - A Comparison](/engineering-education/node-vs-python/)
+
+### Create an NPM Account
+
+To start up, let us create an NPM account.
+
+- Go to the [npmjs website](https://www.npmjs.com/) and click on the **sign up** button.
+
+![npmjs Webpage](/engineering-education/npm-packages/npmjs-homepage.png)
+
+- Enter a username, email address and password.
+
+![Create NPM Account](/engineering-education/npm-packages/npm-signup.png)
+
+- Now, you have to verify your account. Check your email ID that you provided and click on the verification email. This will verify your npm account.
+
+- You have successfully created an NPM account, and can start publishing packages.
+
+- You need to sign in to your npm account in your console to start publishing packages.
+
+- To do this, open a new folder and start your terminal. Enter the command,
+
+```bash
+npm login
+```
+
+- Now, enter your username, password and email ID.
+
+![NPM Login](/engineering-education/npm-packages/npm-login.png)
+
+- We have successfully signed in.
+
+### Let's Code
+
+- Every package published to the NPM registry should have a **`package.json`** file. Let us start by initializing a project.json file.
+
+- The minimum requirements of a `package.json` file is a name and a version. Let us enter those information into the file.
+
+```json
+{
+  "name": "clist-node",
+  "version": "1.0.0"
+}
+```
+
+- In our sample NPM package, we are going to use the axios package for making API calls. Install it by using this command
```

Comment:
You could link to the Axios package npm page here and/or provide further info about what the axios package does.